ui/src/api/api-fetch.ts

```diff
@@ -240,7 +240,7 @@ async function executeBodyRequestInternal<Output>(
         ) {
             return executeBodyRequestInternal(fetchFunction, false);
         } else {
-            if (response.status === 401 && response.headers.has('session-expired')) Snackbar.sessionExpired();
+            if (response.status === 401 && response.headers.has('session-expired') || errorResponse.response.localizedMessageKey === TOKEN_EXPIRED) Snackbar.sessionExpired();
```

Comment:
Noi or:in ehdoti voisi pistää `()`:n sisälle (eli siis näin):

`response.status === 401 && (response.headers.has('session-expired') || errorResponse.response.localizedMessageKey === TOKEN_EXPIRED)`, niin saadaan presedenssit oikein. Muuten tuo localizedMessageKeyn tutkimisen ehto yliajaa tuon alun `response.status === 401`:n

Reply:
juu, korjaan.